<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="ff2dcacf-56f3-4c18-b786-a736d1b9dbd2" name="Mats@MATSDATOR1 2012-01-08 11:07:27" runUser="MatsDator1\Mats" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="8f4e77c5-bbe4-405a-a0f8-8a80e26343c7">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Mats_MATSDATOR1 2012-01-08 11_07_27">
      <DeploymentItem filename="C:\Development\SimpleIOC\SimpleIOC.Core\bin\Debug\SimpleIOC.Core.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-01-08T11:07:27.5976742+01:00" queuing="2012-01-08T11:07:28.7747415+01:00" start="2012-01-08T11:07:28.9607522+01:00" finish="2012-01-08T11:09:14.7768045+01:00" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="MATSDATOR1" outcome="Error" timestamp="2012-01-08T11:09:14.7738043+01:00">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Should_Throw_Exception_When_No_Registration_Exists" storage="c:\development\simpleioc\simpleioc.tests\bin\debug\simpleioc.tests.dll" id="f375579f-1fd7-3809-8422-0bf706c6e23b">
      <Execution id="d4ee4d54-244f-4129-9346-1e007f88d245" />
      <TestMethod codeBase="C:/Development/SimpleIOC/SimpleIOC.Tests/bin/Debug/SimpleIOC.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="SimpleIOC.Tests.ContainerTests, SimpleIOC.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Should_Throw_Exception_When_No_Registration_Exists" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="f375579f-1fd7-3809-8422-0bf706c6e23b" executionId="d4ee4d54-244f-4129-9346-1e007f88d245" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="d4ee4d54-244f-4129-9346-1e007f88d245" testId="f375579f-1fd7-3809-8422-0bf706c6e23b" testName="Should_Throw_Exception_When_No_Registration_Exists" computerName="MATSDATOR1" duration="00:01:45.6680439" startTime="2012-01-08T11:07:29.1087606+01:00" endTime="2012-01-08T11:09:14.7768045+01:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="d4ee4d54-244f-4129-9346-1e007f88d245">
    </UnitTestResult>
  </Results>
</TestRun>